core/src/main/kotlin/behavior/channel/GuildMessageChannelBehavior.kt

```diff
@@ -17,15 +17,31 @@ import kotlin.time.Duration.Companion.days
  */
 public interface GuildMessageChannelBehavior : GuildChannelBehavior, MessageChannelBehavior {
 
-
     /**
      * Requests to bulk delete the [messages].
      * Messages older than 14 days will be deleted individually.
      *
      * @param reason the reason showing up in the audit log
      * @throws [RestRequestException] if something went wrong during the request.
      */
+    @Deprecated("Binary compatibility", level = DeprecationLevel.HIDDEN)
     public suspend fun bulkDelete(messages: Iterable<Snowflake>, reason: String? = null) {
+        bulkDelete(messages, deleteOlderMessages = true, reason)
+    }
+
+    /**
+     * Requests to bulk delete the [messages].
+     * Messages older than 14 days will be deleted individually.
+     *
+     * @param reason the reason showing up in the audit log
+     * @param deleteOlderMessages whether messages from two weeks ago will be deleted
+     * @throws [RestRequestException] if something went wrong during the request.
+     */
```

Comment:
```suggestion
    /**
     * Requests to bulk delete the [messages].
     *
     * If [deleteOlderMessages] is `true` (default), messages older than 14
     * days will be deleted individually. Otherwise they will be ignored.
     *
     * @param reason the reason showing up in the audit log
     * @throws [RestRequestException] if something went wrong during the request.
     */
```
Include docs for the param in the text.